dsvjetl/uhp-whos-your-governor
# Feature/ui setup

### TASK

- Add normalize.css
- Add SCSS variables/mixins/colors
- Material / Element / Vuetify - Or custom?
- Find and implement font-face
- Implement headings (fluid?)
- https://element.eleme.io/#/en-US/ - May be the best?

https://trello.com/c/rVXVp0SF/2-ui-setup

### CHANGES

- [x] Added initial UI/SCSS architecture
- [x] Added Element UI
- [x] Implemented SCSS variables (colors/fonts)
- [x] Implemented fluid headings

### QA

- Make sure there is no style or setup conflicts between various libraries and configurations.
